From 153765a7e873efae083e5cfab826c44803439180 Mon Sep 17 00:00:00 2001
From: TheGreatMcPain <james@thegreatmcpain.xyz>
Date: Fri, 18 Nov 2022 14:43:31 -0600
Subject: [PATCH] Make '/tmp/hypr' readable/writable by everyone.

Definitely not secure, but it fixes issues when using the greetd display
manager.
---
 src/main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.cpp b/src/main.cpp
index 122127f..2ce9e87 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -36,7 +36,7 @@ int main(int argc, char** argv) {
         }
     }
 
-    system("mkdir -p /tmp/hypr");
+    system("mkdir -m777 -p /tmp/hypr");
 
     if (!ignoreSudo) {
         if (Init::isSudo()) {
-- 
2.37.4

